libkatana_python_native/src/EntityTypeManager.cpp

```diff
@@ -83,7 +95,18 @@ katana::python::InitEntityTypeManager(py::module_& m) {
           py::arg("sub_type"), py::arg("super_type"))
       .def(
           "add_atomic_entity_type",
-          &katana::EntityTypeManager::AddAtomicEntityType)
+          [](katana::EntityTypeManager* self,
+             const std::string& name) -> Result<EntityType> {
+            return EntityType{
+                self, KATANA_CHECKED(self->AddAtomicEntityType(name))};
+          })
+      .def(
+          "get_or_add_atomic_entity_type",
+          [](katana::EntityTypeManager* self,
+             const std::string& name) -> Result<EntityType> {
+            return EntityType{
+                self, KATANA_CHECKED(self->GetOrAddEntityTypeID(name))};
+          })
```

Comment:
I think I may have wrapped the same method and called it in the enterprise repo. Oops.

Reply:
Oh to clarify, if you rebase you can see that I wrapped the same method in the open repo and just called it in the enterprise repo.

Reply:
Ah. OK. I'll rebase and handle the difference. It is odd that github thinks this is mergeable though.

Reply:
It's because I named my method differently :)